### PR TITLE
Fix avatar issue before login

### PR DESCRIPTION
**Pull request by sniehus**

### CODE REVIEW

File: src/client/components/menu/Menu.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ const Menu: React.FC = () => {
     setStateSession(sessionID);
     axios.get(`user/find/${sessionID}`)
       .then((result) => {
```
Comment (copelka) [inline src/client/components/menu/Menu.tsx:31]:
```suggestion
      .then(({ data }) => {
```

File: src/client/components/menu/Menu.tsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ const Menu: React.FC = () => {
...
-        console.log('USER', result);
+        //console.log('USER', result);
         const { google_avatar } = result.data
         setActiveUser(result.data);
```
Comment (copelka) [inline src/client/components/menu/Menu.tsx:34]:
```suggestion
        const { google_avatar } = data;
        setActiveUser(data);
```

File: src/client/components/nav/NavBar.tsx [hunk 3]
```diff
@@ -27,8 +27,7 @@ const Nav: React.FC = () => {
     }, 1000);
     return () => clearInterval(interval);
   };
-
-
+  const googleAvatar = avatar ? avatar : 'https://res.cloudinary.com/de0mhjdfg/image/upload/v1676696909/gnawlinzIcons/noun-profile-1094753_lwnwm4.png'
```
Comment (copelka) [inline src/client/components/nav/NavBar.tsx:30]:
```suggestion
  const defaultAvatar = 'https://res.cloudinary.com/de0mhjdfg/image/upload/v1676696909/gnawlinzIcons/noun-profile-1094753_lwnwm4.png';
  const googleAvatar = avatar ? avatar : defaultAvatar;
```

**Review by copelka — CHANGES_REQUESTED** [review]

**Review by MilesAger — APPROVED** [review]
Looks good. Changes could be applied.

**Review by Musique243 — APPROVED** [review]

**Review by MilesAger — APPROVED** [review]
good!

**Review by Musique243 — APPROVED** [review]

**Review by copelka — APPROVED** [review]

**Pull request merged**